.gitattributes

```diff
@@ -0,0 +1 @@
+*.go text eol=lf
```

Comment:
Do we really need this?

Reply:
Yes, we do, I tried without it and the lint was failing on Windows with non-sense complaints on our license headers.

Reply:
🤷 weird, but ok.